examples/truffle-migrate/README.md

```diff
@@ -1,14 +1,22 @@
 # Sample ZeppelinOS project with truffle migrations
 
-This is a sample ZeppelinOS project that runs `zos` from truffle migrations, using the CLI javascript interface. This allows any project that currently relies on truffle migrations to switch over to `zos`, thus deploying upgradeable instances instead of regular ones. Also, by using the CLI interface, `zos` keeps track of all deployed logic contracts and all upgradeable instances, and also tracks them in truffle build artifacts.
+This is a sample ZeppelinOS project that runs `zos` from truffle migrations, using the CLI javascript interface. 
```

Comment:
Have these new lines been added deliberately? I would revert these in favor of not adding new lines after a certain number of words... but as we don't have any convention defined, I'm ok leaving this as it is..

Reply:
No, that was part of an update in v2.1.0